src/pages/QRScanner/index.tsx

```diff
@@ -33,8 +31,8 @@ const QRScanner = () => {
   const [memo, setMemo] = useState("");
   const [LNURLdata, setLNURLdata] = useState(null);
   const [scanQR, setScanQR] = useState(true);
-  const [done, setDone] = useState(null);
-  const [error, setError] = useState(null);
+  const [, setDone] = useState(null);
```

Comment:
if these states are unused, can you remove them entirely just to get rid of any additional unnecessary re-renders?